src/features/auto-resize-grid/js/auto-resize.js

```diff
@@ -41,6 +41,7 @@
             if (newGridHeight !== prevGridHeight || newGridWidth !== prevGridWidth) {
               uiGridCtrl.grid.gridHeight = newGridHeight;
               uiGridCtrl.grid.gridWidth = newGridWidth;
+              uiGridCtrl.grid.api.core.raise.gridHeightChanged(prevGridHeight, newGridHeight);
```

Comment:
gridDimensionChanged does not exist already, correct?  I should just update the event I made here with this?